package.json

```diff
@@ -164,6 +165,7 @@
   },
   "devDependencies": {
     "@elastic/eslint-config-kibana": "0.0.3",
+    "accept-language-parser": "1.1.2",
```

Comment:
There's shouldn't be separate lines for this module in dependencies and devDependencies.